src/components/organisms/RecipesContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import { CircularProgress, styled } from "@mui/material";
+import { useFetchRecipes } from "../../api/recipes";
+import { useAppDispatch } from "../../store/store";
+import { showSnackbar } from "../../slices/snackbarSlice";
+import { useTranslation } from "react-i18next";
+import RecipesList from "../molecules/RecipesList";
+
+const ProgressContainer = styled("div")({
+  width: "100%",
+  height: "100%",
+  display: "flex",
+  justifyContent: "center",
+  alignItems: "center",
+});
+
+const RecipeContainer = () => {
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation();
+  const recipesQuery = useFetchRecipes();
```

Comment:
a better way would be to do something like this: `{data}=useFetchRecipes` you can also extract other useful things like isLoading flag or object with errors